lessons/intro/distribution/index.md

```diff
@@ -74,89 +80,89 @@ Je to dobrá konvence, ale není to technicky nutné.
+a vytvoří zdrojovou a binární distribuci.
 
 ```console
-(__venv__)$ python setup.py sdist
+(__venv__)$ python -m pip install build
 ...
-warning: sdist: standard file not found: should have one of README, README.rst, README.txt
+(__venv__)$ python -m build
+* Creating venv isolated environment...
+* Installing packages in isolated environment... (setuptools>=61.0)
+* Getting build dependencies for sdist...
 ...
+Successfully built isholiday-0.1.0.tar.gz and isholiday-0.1.0-py3-none-any.whl
 ```
 
+Všimněte si nového adresáře v projektu: `dist`, v němž jsou dva archivy:
+- `isholiday-0.1.0.tar.gz` - zdrojový balíček
+- `isholiday-0.1.0-py3-none-any.whl` - binární balíček (tzv. _wheel_). 
+
+Můžete je rozbalit pomocí systémových nástrojů a podívat se dovnitř.
+
+> [note]
+> `whl` je ve skutečnosti stejný formát jako `zip`,
+> takže pokud vaše systémové nástroje ho neumí detekovat,
+> stačí přejmenovat rozšíření souboru na `.zip`.
+
+Můžete také vytvořit pouze zdrojový nebo pouze binární balíček pomocí přepínačů `--sdist`, resp. `--wheel`.
+
+
 Extra soubory do zdrojového balíčku
 -----------------------------------
 
-Jak vidíte, `setuptools` si stěžuje, že náš projekt nemá `README` – soubor,
-do kterého se tradičně píšou základní informace o projektu.
+Náš projekt nemá `README` – soubor, do kterého se tradičně píšou základní informace o projektu.
 Můžeme jej vytvořit a uložit jako `README` přímo v kořenovém adresáři projektu,
 tedy tam, kde byste jej nejspíš čekali.
 
 ```
 Czech public holiday checker...
 ```
 
-Poté spustíme `setup.py sdist` znovu:
-
-```console
-(__venv__)$ python setup.py sdist
-```
-
-V adresáři `dist` najdete archiv, jeho obsah můžete zkontrolovat. Měl by tam
-být i soubor `README`.
-
-Skvělé, pojďme vytvořit i další speciální soubor, `LICENSE`, který bude
+Pojďme vytvořit i další speciální soubor, `LICENSE`, který bude
 obsahovat text licence, v tomto případě Public Domain.
 Obsah najdete třeba na [CC0].
 
 [CC0]: https://creativecommons.org/publicdomain/zero/1.0/legalcode.txt
 
-Pokud ale se souborem `LICENSE` vytvoříte zdrojový balíček, soubor v archivu
-nebude. Je to proto, že se standardně do archivu přidávají jen některé soubory.
-Další soubory lze přidat pomocí souboru `MANIFEST.in`, dle [dokumentace].
-
-[dokumentace]: https://docs.python.org/3/distutils/sourcedist.html#specifying-the-files-to-distribute
-
-V našem případě bude `MANIFEST.in` vypadat takto:
-
-```
-include LICENSE
-```
-
-Při dalším spuštění už `setup.py` přidá i soubor `LICENSE`.
-To můžete zkontrolovat i ve výsledném archivu.
+Poté vytvoříme balíčky znovu:
 
 ```console
-(__venv__)$ python setup.py sdist
-...
-hard linking LICENSE -> isholiday-0.1
-hard linking MANIFEST.in -> isholiday-0.1
-hard linking README -> isholiday-0.1
-...
+(__venv__)$ python -m build
 ```
 
+Zkontrolujte obsah archivu, který najdete v adresáři `dist`
+Měly by tam být oba soubory.
```

Comment:
Podle mě README ve wheelu nebude (ale možná se pletu). A LICENSE bude v dist infu, kde to nikdo nebude hledat.